supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-enterprise-linux.adoc

```diff
@@ -35,6 +456,149 @@ For documentation questions, contact rhel-docs@redhat.com.
+* `hardened` for passwords hardened against brute-force attempts
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[kerberos-keytab]]
+==== image:images/yes.png[yes] Kerberos keytab (noun)
+*Description*: While a password is the default authentication method for a user, keytabs are the default authentication method for hosts and services. A Kerberos keytab is a file that contains a list of Kerberos principals and their associated encryption keys, so a service can retrieve its own Kerberos key and verify a user’s identity. For example, every IdM client has an `/etc/krb5.keytab` file that stores information about the `host` principal, which represents the client machine in the Kerberos realm.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:kerberos-principal[Kerberos principal]
+
+[discrete]
+[[kerberos-principal]]
+==== image:images/yes.png[yes] Kerberos principal (noun)
+*Description*:  Unique Kerberos principals identify each user, service, and host in a Kerberos realm. Kerberos principals adhere to the following naming conventions:
+
+* For users: `identifier@REALM`, such as `admin@EXAMPLE.COM`.
+* For services: `service/fully-qualified-hostname@REALM`, such as `http/server.example.com@EXAMPLE.COM`
+* For hosts: `host/fully-qualified-hostname@REALM` such as `host/client.example.com@EXAMPLE.COM`
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:kerberos-realm[Kerberos realm]
+
+[discrete]
+[[kerberos-protocol]]
+==== image:images/yes.png[yes] Kerberos protocol (noun)
+*Description*: Kerberos is a network authentication protocol that provides strong authentication for client and server applications by using secret-key cryptography. IdM and Active Directory use Kerberos for authenticating users, hosts and services.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[kerberos-realm]]
+==== image:images/yes.png[yes] Kerberos realm (noun)
+*Description*: A Kerberos realm encompasses all the principals managed by a Kerberos Key Distribution Center (KDC). In an IdM deployment, the Kerberos realm includes all IdM users, hosts, and services.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:idm-deployment[IdM deployment], xref:key-distribution-center[Key Distribution Center]
+
+[discrete]
+[[kerberos-ticket-policies]]
+==== image:images/yes.png[yes] Kerberos ticket policies (noun)
+*Description*: The Kerberos Key Distribution Center (KDC) enforces ticket access control through connection policies, and manages the duration of Kerberos tickets through ticket lifecycle policies. For example, the default global ticket lifetime is one day, and the default global maximum renewal age is one week.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:key-distribution-center[Key Distribution Center]
+
+[discrete]
+[[key-distribution-center]]
+==== image:images/yes.png[yes] Key Distribution Center (KDC) (noun)
```

Comment:
Can you remove the (KDC) in the heading here? Usually we want what it is shortened to in the description below

Reply:
+1